python/ray/_private/usage/usage_lib.py

```diff
@@ -356,6 +357,9 @@ def _generate_cluster_metadata():
                 "session_start_timestamp_ms": int(time.time() * 1000),
             }
         )
+        if sys.platform == "linux":
+            # Record glibc version
+            metadata.update({"glibc": platform.libc_ver()[1]})
```

Comment:
You need to use `record_extra_usage_tag` otherwise we need to update the schema.

Reply:
Which one do you think should be a system flag and which one should be extra usage tag?

Reply:
Good question. We currently don't have a guideline for it. Currently extra usage tags are used by libraries to record their own usage stats so they are more like optional ones.

I think it's fine to be a system flag, I just need to update the schema and redeploy the server.

Reply:
Let me know if it's hard and I can update it to use tags.
Should we make all of them as a flexible schema? I feel it's more convenient.